### PR TITLE
Created Header with link on rules and styled it

### PR DESCRIPTION
**Pull request by camillaume**

### CODE REVIEW

**Review by ValentinCop — DISMISSED** [review]

**Review by LuigiMenez — DISMISSED** [review]
Ok

**Review by LoicBrassart — DISMISSED** [review]
Saw no Evil

**Review by ValentinCop — DISMISSED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Review by LoicBrassart — DISMISSED** [review]
SnE

**Review by LuigiMenez — DISMISSED** [review]
Ok

**Review by ValentinCop — DISMISSED** [review]

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -1,32 +1,30 @@
...
           <Route path="/legal" element={<Legal />} />
           <Route path="/game" element={<Game />} />
           <Route path="/rules" element={<Rules />} />
-          
             </Routes>
-
         </main>
-      </Styled>
       <Footer />
+             </Styled>
```
Comment (LuigiMenez) [inline frontend/src/App.jsx]: Attention to indentation

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Verified the configuration of prettier for indentation

**Review by ValentinCop — DISMISSED** [review]

**Review by ValentinCop — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me

**Review by LoicBrassart — APPROVED** [review]
No more Evil

**Pull request merged**